src/js/blight/BlightContainer.jsx

```diff
@@ -1,10 +1,41 @@
 import React, { Component } from "react";
 
 export default class BlightContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.totalBlight = this.props.players * 2;
+
+    this.state = { value: 10 };
+
+    this.handleClickP = this.handleClickP.bind(this);
+    this.handleClickM = this.handleClickM.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleClickP(e) {
```

Comment:
Try not to abbreviate class/function/variable names. 
Also try to give descriptive names. For simple components, it's okay to have some generic `handleClick` handlers, but for more complex ones, like this one with 2 handlers, you should try to distinguish them. 
Maybe `handleAddBlight` and `handleRemoveBlight`